lib/suse/connect/api.rb

```diff
@@ -47,6 +47,7 @@ def make_request(method, url, options)
         options[:userpwd] = "#{@username}:#{@password}" unless options[:userpwd]
         options[:method] = method
         options[:accept_encoding] = 'gzip, deflate'
+        options[:headers] = { 'RMT' => @username }
```

Comment:
Aren't those organization credentials? The machine credentials should be read from `/etc/zypp/credentials.d/SCCcredentials`.

Reply:
yep, that was exactly what I meant on stand up. many thanks for help @ikapelyukhin :+1: